packages/sources/tiingo/src/adapter.ts

```diff
@@ -59,42 +59,41 @@ interface UpdateMessage extends Message {
 }
 
 const customParams = {
-  base: ['base', 'from', 'coin'],
-  quote: ['quote', 'to', 'market'],
+  base: ['base', 'from', 'coin', 'ticker'],
 }
 
 export const makeWSHandler = (config?: Config): MakeWSHandler | undefined => {
   if ((process.env.NODE_ENV || '').toLowerCase() !== 'development') {
     return undefined
   }
 
-  const getSubscription = (pair: string | undefined, subscribe = true) => {
+  const getSubscription = (ticker: string | undefined, subscribe = true) => {
     const defaultConfig = config || makeConfig()
-    if (!pair) return
+    if (!ticker) return
     return {
       eventName: subscribe ? 'subscribe' : 'unsubscribe',
       authorization: defaultConfig?.apiKey,
       eventData: {
         thresholdLevel: 5, // only Last Trade updates
-        tickers: [pair],
+        tickers: [ticker],
       },
     }
   }
-  const getPair = (input: AdapterRequest) => {
+  const getTicker = (input: AdapterRequest): string | undefined => {
     const validator = new Validator(input, customParams, {}, false)
     if (validator.error) return
     const base = validator.validated.data.base.toLowerCase()
-    const quote = validator.validated.data.quote.toLowerCase()
-    return `${base}${quote}`
+    return base
   }
   return () => {
     const defaultConfig = config || makeConfig()
     return {
       connection: {
         url: defaultConfig.api.baseWsURL || DEFAULT_WS_API_ENDPOINT,
       },
-      subscribe: (input) => getSubscription(getPair(input)),
-      unsubscribe: (input) => getSubscription(getPair(input), false),
+      shouldNotServeInputUsingWS: (input) => input.data.endpoint !== 'iex',
```

Comment:
We should also accept any of the aliases for iex. In other words, any entry in this array: https://github.com/smartcontractkit/external-adapters-js/blob/4e34ac7a144de024dcc45a62acddbf5f5fd92826/packages/sources/tiingo/src/endpoint/iex.ts#L4